src/app/templates/ts/src/index.test.ts

```diff
@@ -0,0 +1,31 @@
+import <%= pluginFunctionName %> from "./index"
+
+declare const global: any
+
+describe("<%= pluginFunctionName %>()", () => {
+  beforeEach(() => {
+    global.warn = jest.fn()
+    global.message = jest.fn()
+    global.fail = jest.fn()
+    global.markdown = jest.fn()
+  })
+
+  afterEach(() => {
+    global.warn = undefined
+    global.message = undefined
+    global.fail = undefined
+    global.markdown = undefined
+  })
+
+  it('Checks for a that message has been called', () => {
+    global.danger = {
+      github: { pr: { title: 'My Test Title' } },
+    }
+
+    <%= pluginFunctionName %>()
+
+    expect(global.message).toHaveBeenCalledWith(
+      'PR Title: My Test Title',
```

Comment:
> I created a TypeScript plugin and opened it in VS Code. This line caused a lint error since it uses single quotes and `tslint.json` require double quotes.